src/modules/Interactive.js

```diff
@@ -0,0 +1,14 @@
+const ChangeTaskStatus = (arr, id) => {
+  if (arr[id - 1].completed === true) {
+    arr[id - 1].completed = false;
+  } else {
+    arr[id - 1].completed = true;
+  }
+  return arr;
+};
+
+const clearList = (arr) => {
+  const filtered = arr.filter((element) => element.completed === false);
```

Comment:
- [x] Good job filtering your array, however, I think it would be better to use `!element.completed` instead of `element.completed === false` since it is the same and it helps you keep your code simple (KISS).